mappings/net/minecraft/block/entity/BlockEntityType.mapping

```diff
@@ -17,7 +17,13 @@ CLASS net/minecraft/class_2591 net/minecraft/block/entity/BlockEntityType
 		FIELD field_11915 supplier Lnet/minecraft/class_2591$class_5559;
 		FIELD field_19316 blocks Ljava/util/Set;
 		METHOD <init> (Lnet/minecraft/class_2591$class_5559;Ljava/util/Set;)V
+			ARG 1 creator
```

Comment:
```suggestion
			ARG 1 factory
```
Seems like you missed a param 😅

Reply:
Woops, thanks. I've also updated the other `supplier` names to `factory` too.